man/spark_read_json.Rd

```diff
@@ -14,6 +14,8 @@ spark_read_json(sc, name, path, repartition = 0, memory = TRUE,
 
 \item{path}{The path to the file. Needs to be accessible from the cluster. Supports: "hdfs://" or "s3n://"}
 
+\item{options}{A list of strings with additional options.}
```

Comment:
We need to add a hyperlink here to whatever docs Spark provides on what options are available.